code/modules/reagents/reagent_dispenser.dm

```diff
@@ -142,6 +142,39 @@
 	explosion(src, devastation_range = 1, heavy_impact_range = 2, light_impact_range = 7, flame_range = 12)
 	qdel(src)
 
+/obj/structure/reagent_dispensers/cream_soda //too fizzy, too furious
+	name = "cream soda barrel"
+	desc = "Be careful with that! That cream soda is serious business!"
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "barrel"
+	reagent_id = /datum/reagent/consumable/cream_soda
+	can_be_tanked = FALSE
+
+/obj/structure/reagent_dispensers/cream_soda/boom()
+	explosion(src, heavy_impact_range = 1, light_impact_range = 5, flame_range = 0)
+	qdel(src)
+
+/obj/structure/reagent_dispensers/cream_soda/blob_act(obj/structure/blob/B)
```

Comment:
Descriptive var names, please!

Reply:
damn you copy paste, you've ruined me with your old variable names

Reply:
Yes

Reply:
…/the_blob would work or something

Reply:
or just blob